README.md

```diff
@@ -0,0 +1,12 @@
+# OpenTTD's Automated Team Signup
+
+To be part of a team in OpenTTD, simply create an issue in this repository based on the template of the team you want access to.
+After approval from one of the Core Developers, you will be added to that team.
+
+## TODO list
+
+.dorpsgek.yml
+Workflows
+
+- Every night, check OpenTTD src/lang for any new languages
+  - If found, create the required template file, and make a pull-request
```

Comment:
But don't create a PR if one is already open?
IMO not worth the effort, adding a language requires adding a team and commits in various repositories.

Anyway, are you aware of
http://translator.openttd.org/language-list
This URL is meant for scripts to query languages.